### PR TITLE
C hbranch

### PR DESCRIPTION
**Pull request by NiarH**

### CODE REVIEW

File: source/guide/3.1/installation-manual.md [hunk 1]
```diff
@@ -46,3 +46,4 @@ For more information on the packages, please refer to the [Cloudify installation
 You can now direct your browser at http://manager-ip-address to access the UI or run `cfy use manager-ip-address` (depending on what interface you're using).
 
 Enjoy!
+uju
```
Comment (uric) [inline source/guide/3.1/installation-manual.md]: @NiarH woot??

File: source/guide/3.1/quickstart.md [hunk 2]
```diff
@@ -269,6 +269,6 @@ vagrant box remove cloudify-box
 
 # What's Next
 
-* Understand the requirements for bootstrapping your very own Cloudify Manager by reading [this]({{page.installation_general_link}}).
+* Understand the requirements for bootstrapping your very own Cloudify Manager by reading under [Installation]({{page.installation_general_link}}).
 * Try to install the same application on OpenStack by following the [OpenStack Tutorial]({{page.quickstart_openstack_link}}).
```
Comment (uric) [inline source/guide/3.1/quickstart.md]: @NiarH reading under is not proper English...

**Comment by uric** [comment]
Closing this

**Comment by NiarH** [comment]
Sorry Uri we were merely testing it for the first time, anyway, the above rewording was meant to correct the link reference replacing the word "this" and, as far as i know, "reading under <link>" is a common usage, perhaps you can suggest a better alternative? Thx

**Pull request closed without merging**